src/mbgl/mtl/offscreen_texture.cpp

```diff
@@ -38,6 +38,8 @@ class OffscreenTextureResource final : public RenderableResource {
         commandBuffer->commit();
```

Comment:
Could you please revert changes in this file, I already cached the render descriptor here and done more changes, so to avoid conflicts.